cli.js

```diff
@@ -0,0 +1,65 @@
+// requirements
+var Word = require('./word.js');
+var inquirer = require('inquirer');
+
+console.log("Time to play Hangman\nLetter by letter guess name of the great house of Westeros\n\n***********************************************************");
+
+var Game = function(wordBank, wordsRight, guessesLeft, currentHouse) {
+	this.wordBank = ["lannister", "stark", "baratheon", "targaryen", "tully", "tyrell", "arryn", "greyjoy", "martell" ],
+	this.wordsRight = 0,
+	this.guessesLeft = 10,
+	this.currentHouse = null,
+
+	this.startGame = function (house) {
+		this.resetGuesses();
+		this.currentHouse = new Word(this.wordBank[Math.floor(Math.random()* this.wordBank.length)]);
+		this.currentHouse.getLetter();
+		this.promptUser();
+	},
+
+	this.resetGuesses = function() {
+		this.guessesLeft = 10;
+	},
+
+	this.promptUser = function() {
+		var user = this;
+		inquirer.prompt([
+			{
+				name: "guessedLetter",
+				message: "enter your guess",
+				type: "input"
+			}
+			]).then(function(answer) {
+				var guessedLetter = new guessLetter(answer.guessLetter);
```

Comment:
This isn't referring to any constructor that I have observed in your code. This could be dangerous as the outcome can vary. Instead I would just store the value of answer.guessLetter and not use the new keyword.